packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerServiceOptions.ts

```diff
@@ -101,7 +101,10 @@ export type OpenId4VciCredentialRequestToCredentialMapper = (options: {
   credentialsSupported: OpenId4VciCredentialSupported[]
 }) => Promise<OpenId4VciSignCredential> | OpenId4VciSignCredential
 
-export type OpenId4VciSignCredential = OpenId4VciSignSdJwtCredential | OpenId4VciSignW3cCredential
+export type OpenId4VciSignCredential = { credentialSupportedId: string } & (
```

Comment:
Now if you import the `OpenId4VciSignSdJwtCredential` it doesn't include the `credentialSupportedId`. I think either we need a base that `OpenId4VciSignSdJwtCredential` and `OpenId4VciSignW3cCredential` extend from, or we should do a bit of duplication